.github/workflows/pr-preview.yml

```
@@ -0,0 +1,162 @@
+            next
+          }
+          { print }
+          ' fly.toml > fly.toml.tmp && mv fly.toml.tmp fly.toml
+          
+          # Copy and modify litefs.yml for PR preview (standalone, no consul)
+          cp other/litefs.yml other/litefs.yml.backup
+          
+          # Replace consul lease with static lease for PR preview isolation
+          # This makes each PR preview a standalone instance
+          sed -i '/^lease:/,/^$/c\
+lease:\
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 64: could not find expected ':'

(syntax-check)